great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -198,18 +182,14 @@ def evaluation_parameter_builder_configs(
     ) -> Optional[List[ParameterBuilderConfig]]:
         return self._evaluation_parameter_builder_configs
 
-    @property
-    def json_serialize(self) -> Union[str, bool]:
-        return self._json_serialize
-
-    @json_serialize.setter
-    def json_serialize(self, value: Union[str, bool]) -> None:
-        self._json_serialize = value
-
     @property
     def fully_qualified_parameter_name(self) -> str:
         return f"{PARAMETER_KEY}{self.name}"
 
+    @property
+    def raw_fully_qualified_parameter_name(self) -> str:
+        return f"{RAW_PARAMETER_KEY}{self.name}"
+
```

Comment:
Requesting docstring to be added to these properties so that more context around `fully_qualified_parameter_name` and `raw_fully_qualified_parameter_name` can be included

Reply:
I'm still having a bit of a challenge coming around to the user of `raw` here. As you said, this is a standard terminology, but I feel the naming could be 

* `raw_fully_qualified_parameter_name`
* `serialized_fully_qualified_parameter_name` ?